<launch>

    <node pkg="augmented_object_detector" name="object_detector" type="object_detector" respawn="false" output="screen">
       <!--topic remapping-->
	<remap from="get_model_mesh_srv" to="/objects_database_node/get_model_mesh" />
	<remap from="get_model_desc_srv" to="/objects_database_node/get_model_description" />
        <remap from="get_model_list_srv" to="/objects_database_node/get_model_list" />
        <!--remap from="cloud_new_in" to="narrow_stereo_textured/points2" /-->
     </node>
        
</launch>

